vuepress/docs/v7.1/docs/reference/entandoapp-cr.md

```diff
@@ -5,7 +5,7 @@ sidebarDepth: 2
 # EntandoApp Custom Resource 
 The EntandoApp CR is the deployment of a Docker image that hosts the Entando and Java-based web application. Server-side components include the Entando App Engine, Entando Component Manager, Entando App Builder, and the user-facing application.
  
-Entando offers standard WildFly, EAP, or Tomcat images for the definition, but typically, customers provide their own CRD
+Entando offers standard WildFly or EAP images for the definition, but typically, customers provide their own CRD
```

Comment:
@jyunmitch for later, we should remove "typically, customers provide their own CRD". We're really trying to discourage that and each release we're eliminating reasons for customers to need to do that.

Reply:
will note this